src/eval/operation.rs

```diff
@@ -465,6 +465,8 @@ fn process_unary_operation(
                         pos_op,
                     )), //TODO include the position of operators on the stack
                 }
+            } else if let Term::ParseError(parse_error) = &*t {
```

Comment:
The problem with this is that Nickel imports aren't necessarily behind `StaticAccess`. Most code examples use a top level record to simulate some notion of a module. But you could put in any other expression.

So if you write `import "lib.ncl"` where:

```nickel
# lib.ncl
"Nickel"$
```

The evaluator will escape this check.

Maybe it's better to check for parse error terms in `nickel_lang::eval::eval_closure` instead, as it contains the main loop. I'm not entirely sure if this will cover all cases. 

Another option would be to refactor the `if let Term ... else { Err ...` pattern to make sure all of them test for `ParseError`. Lots of code editing :P

Everything else LGTM.

Reply:
Thanks for the feedback. It was easy to move the check to `eval_closure`, and that covers both parse error examples. Could you take a look at the latest commit?